templates/assets/css/vlabs-style.css

```diff
@@ -178,10 +180,14 @@ iframe {
 }
 
 .simulation-container {
-  position: relative;
-  overflow: hidden;
-  width: 100%;
-  /* padding-top: 56.25%; */
+  padding: 0px;
```

Comment:
simulation-container houses floating menu button, and iframe. Hence the experiments having position absolute will be scoped within iframe, and thus, will be independent from styling of simulation-container. Hence, it would not create any issues with absolute positioning.